packages/patternfly-4/react-core/src/components/Select/SelectToggle.js

```diff
@@ -0,0 +1,149 @@
+import React, { Component } from 'react';
+import styles from '@patternfly/patternfly-next/components/Select/select.css';
+import { css } from '@patternfly/react-styles';
+import PropTypes from 'prop-types';
+import { CaretDownIcon } from '@patternfly/react-icons';
+
+const propTypes = {
+  /** HTML ID of dropdown toggle */
+  id: PropTypes.string.isRequired,
```

Comment:
The id should not come down from the select here.  I think for now we can just generate it here.  I only see it being referenced by the aria-labeledby on this same div.
@jgiardino I know accessibility docs will be updated once this implementation is done,  should we have this match core for now?

Reply:
Similar to a form field, a label should accompany this type of input.

The way the labels are handled in [core](https://pf4.patternfly.org/components/Select/examples/) is what I'd expect here, but I would also expect the consumer to decide if they want a visible or hidden label. For this implementation, I'm not sure what should be built into the component versus just demonstrated in an example. Is it possible to update the current example and add another example so that we show how you could implement this with both a visible and hidden label?